csi/volumeutils.py

```diff
@@ -478,6 +505,8 @@ def mount_glusterfs(volume, target_path):
     """Mount Glusterfs Volume"""
     if not os.path.exists(target_path):
         makedirs(target_path)
+        # Make mount point immutable if not mounted
+        execute(["chattr", "+i", target_path])
```

Comment:
immutable means no writable? is that fine too?

Reply:
immutable before mount, so that will not be writable when unmounted.

Reply:
Ack!